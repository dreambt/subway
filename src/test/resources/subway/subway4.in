L3-1,L4-7