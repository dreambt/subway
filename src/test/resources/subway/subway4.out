6:L3-1,L3-2,L3-3,X1,L1-3,L1-4,X2,X4,L3-8,L3-9,X8,L4-8,L4-7